charts/microgateway/values.yaml

```diff
@@ -161,18 +161,18 @@ service:
   port: 80
   # service.tlsPort -- Service TLS port
   tlsPort: 443
-  # service.annotations -- Annotations to set on the service
+  # service.annotations -- Annotations to set on the service.
   annotations: {}
     # service.alpha.openshift.io/serving-cert-secret-name: microgateway
-  # service.labels -- Additional labels to add on the service
+  # service.labels -- Additional labels to add on the service.
   labels: {}
 
-# ingress -- [Kubernetes Ingress](#kubernetes-ingress)
+# ingress -- [Kubernetes Ingress](#kubernetes-ingress).
```

Comment:
Remove the dot. 
"Ein- oder Zweiwortbenennungen bekommen keinen Punkt am Ende"

Reply:
Done